pages/docs/running-grouparoo/commands.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: "Grouparoo CLI Commands"
```

Comment:
Based on the way we broke up the sections, I'm now thinking the CLI should be its own section. If we're using sections in the doc as a developer workflow ...

1. Install/Deploy
2. Configure
3. Run

Then the CLI, in general, doesn't fit solely within _Run_. I'd propose we have a separate section that is all about the CLI.